lib/jsdom/living/nodes/HTMLFormElement-impl.js

```diff
@@ -47,14 +47,53 @@ class HTMLFormElementImpl extends HTMLElementImpl {
     super._descendantRemoved.apply(this, arguments);
   }
 
+  _getElementNodes() {
+    if (!this.hasAttributeNS(null, "id")) {
+      return domSymbolTree.treeToArray(this, {
+        filter: node => this._filterElement(node)
+      });
+    }
+
+    const id = this.getAttributeNS(null, "id");
+    return domSymbolTree.treeToArray(this.ownerDocument, {
+      filter: node => {
+        if (!this._filterElement(node)) {
+          return false;
+        }
+
+        if (this.contains(node)) {
```

Comment:
I don't believe this works. In particular `form=""` should take precedent over `contains()` relationships. Another candidate for a test to write, I guess?

Reply:
For this, added:  `test/web-platform-tests/to-upstream/html/semantics/forms/the-form-element/form-elements-form-owner.html`
test: "`form.elements` should not contain elements inside form with different form owner"